src/main/java/it/eng/idsa/businesslogic/configuration/WebConfig.java

```diff
@@ -0,0 +1,16 @@
+package it.eng.idsa.businesslogic.configuration;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+@EnableWebMvc
+public class WebConfig implements WebMvcConfigurer {
```

Comment:
Can this one be replaced in SecurityConfig.java with .cors()?
https://www.baeldung.com/spring-cors#cors-with-spring-security

Reply:
Didn't know about that one. Will do, thanks!